packages/jest-snapshot/src/colors.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// https://jonasjacek.github.io/colors/
+// https://webaim.org/resources/contrastchecker/
+
+// Contrast 6.89 AA
+export const aForeground2 = 90; // #870087
+export const aBackground2 = 225; // #ffd7ff
+
+// Contrast 7.27 AAA
+export const bForeground2 = 22; // #005f00
+export const bBackground2 = 194; // #d7ffd7
+
+export type RGB = [number, number, number];
+
+// Contast 7.36 AAA
+export const aForeground3: RGB = [128, 0, 128]; // #800080
```

Comment:
thoughts on 
```suggestion
export const aForeground3: RGB = [0x80, 0x0, 0x80];
```
instead of the comment?

Reply:
Yes, your wish is my command :)